core/data-transformers/scripts-transformer.ts

```diff
@@ -66,6 +66,7 @@ export function scriptsTransformer(scripts: BigCommerceScripts): C15tScripts {
     const baseConfig: C15tScript = {
       category: mapConsentCategory(script.consentCategory),
       id: script.entityId,
+      target: script.location === 'HEAD' ? 'head' : 'body',
```

Comment:
👏 Awesome that we support this now. Maybe you already mentioned it to Chris on the call we had, but we should include in the release notes since [the last release notes mentioned this feature was coming soon](https://developer.bigcommerce.com/docs/storefront/catalyst/release-notes/1-3-0#consent-manager:~:text=We%E2%80%99re%20collaborating%20closely%20with%20the%20c15t%20team%20to%20extend%20support%20for%20additional%20features%20such%20as%20footer%20script%20placement.%20If%20your%20storefront%20relies%20on%20specific%20script%20placement%20or%20privacy%20handling%20use%20cases%2C%20we%E2%80%99d%20love%20to%20hear%20your%20feedback%20as%20we%20refine%20this%20integration%20further.)